mne_qt_browser/tests/test_pg_specific.py

```diff
@@ -135,16 +135,48 @@ def test_ch_specific_annot(raw_orig, pg_backend):
     # change the stop value of the annotation
     annot_dock.stop_bx.setValue(6)
     annot_dock.stop_bx.editingFinished.emit()
-    # does the channel specific rectangle stay in sync with the annot?
+    # does the single channel annot stay within the annot
     assert annot_dock.stop_bx.value() == 6
-    assert last_fill_between.xData[1] == 6
+    assert single_channel_annot.lower.xData[1] == 6
 
     # now change the start value of the annotation
     annot_dock.start_bx.setValue(4)
     annot_dock.start_bx.editingFinished.emit()
     # does the channel specific rectangle stay in sync with the annot?
     assert annot_dock.start_bx.value() == 4
-    assert last_fill_between.xData[0] == 4
+    assert single_channel_annot.lower.xData[0] == 4
+
+    ch_index = np.mean(annot.single_channel_annots["MEG 0133"].ypos).astype(int)
+
+    # MNE >= 1.8
+    if check_version("mne", "1.8"):
+        # test if shift click an existing annotation removes object
+        fig._fake_click(
+            (4 + 2 / 2, ch_index),
+            xform="data",
+            button=1,
+            modifier=Qt.ShiftModifier,
```

Comment:
Rather than having to import `Qt` here, this would be nicer as:
```suggestion
            modifier="shift",
```
then inside `_fake_click` you can do something like
```
if modifier is not None:
    modifier = getattr(Qt, f"{modifier.capitalize()}Modifier}")
```
That also makes things future compatible with the matplotlib backend where eventually we might want the `_fake_click` there to support modifiers as well and have it work for both browsers.

Reply:
Do you want to have modifiers as strings? I can try but it would be a bit of work. The `_fake_click()` function can call `_mouseClick()`, `_mouseMove()`, `_mouseRelease()` and `_mouseDrag()`. All of those, and some more functions, would have to be altered as they all accept modifier arguments. I could create a function that takes in a string or modifier and outputs the modifier. But still may be some work and for another PR. 

Other than that do you think this branch is ready to be merged?

Reply:
Yeah we can change it as a follow-up PR I think, so I'll go ahead and merge -- thanks!